__tests__/topics.test.js

```diff
@@ -0,0 +1,32 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topic objects to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(Array.isArray(response.body)).toBe(true);
+        expect(response.body.length).toBe(3);
+        response.body.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
+  test("GET: 404 responds with an error when route is non existant", () => {
```

Comment:
This is a valid test, but is actually testing all bad URLs, not just ones associated with /api/topics. With this in mind, i think it should be in it's own describe block not in /api/topic desc